style/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+    font-size: 16px;
+    background-color: rgb(255, 255, 255);
+    width: 100%;
```

Comment:
ogolnie nie mozna ustawiac max width czy width na body, lepiej bylo by to ogarnac "containerem" o ktorym mowilismy na zajeciach